### ISSUE TITLE
3.5 Form Checkbox bug  

### ISSUE THREAD

**Issue by rossie70** [dotCMS/core#9477]
There may be a bug with the checkbox field type in 3.5+. I tested it on demo.dotcms.com and was able to reproduce the problem.

On the canned "Contact Us" form, I added a checkbox field "How can we contact you?" with 4 options. I then submitted the form from the front-end, choosing two of the options. When I view the saved content on the back-end, the form submission was saved as though all 4 options were selected.

**Comment by john-thomas-dotcms** [comment]
@rossie70, I tried this out on demo.dotcms.com and wasn't able to reproduce it - which probably means I'm missing a step somewhere.

If you have a chance, can you please outline the exact steps you used to reproduce this - from adding the field (what is the field value, what options were checked on the field, etc.) to how you accessed the form and what values you entered on the front-end?  Note: Feel free to use [gist](https://gist.github.com/) to upload screen shots or other info.

**Comment by rossie70** [comment]
@john-thomas-dotcms, Jere is the new checkbox field added:

![2016-08-01 17_05_17-dotcms _ enterprise web content management](https://cloud.githubusercontent.com/assets/3037083/17484228/2455d1f6-5d46-11e6-8966-29687cf2902c.png)

Front end form:
![contact](https://cloud.githubusercontent.com/assets/3037083/17484273/4cec3754-5d46-11e6-8fd2-5e7b33e3d5e4.png)

View of the saved submission on the back-end:
![2016-08-01 17_10_08-dotcms _ enterprise web content management](https://cloud.githubusercontent.com/assets/3037083/17484291/642e454c-5d46-11e6-809c-4e03706d4b76.png)

**Comment by john-thomas-dotcms** [comment]
@rossie70, I tried again, and was again not able to reproduce the problem.  I tried in Chrome, Firefox and IE 11 under Windows 7.  So there's a few more things we need to look at to see if we can track down the problem.

1. Make sure you're not trying to access the backend of multiple dotCMS sites at the same time in the same browser.
    * You can only login to the backend of 1 dotCMS site at a time; if you login to multiple sites at the same time, you can see some really odd problems.
    * Note that it is possible to access multiple dotCMS backends from the same browser without problems, but only if you open each new dotCMS site in an Incognito window (which creates a new session).
        * So the simplest thing to do to see if this is the problem, is open a new incognito window in your browser and try again.
2. If you're still seeing the problem, can you please clear your browser cache, and then close and reopen your browser?
    * Alternately (if you really don't want to clear your cache), try running on another machine which has never accessed the dotCMS backend before.
    * The bottom line is that you may have some old versions of the controls or libraries in your browser cache, which could cause unexpected behavior.
3. If that doesn't solve the problem, can you please try the same thing using a different browser?
    * As I said, I tried Chrome, Firefox & IE 11.  See if one of these works for you.
4. If none of that works, can you let me know what OS, OS version, browser, browser version, and Java version you're using?

**Comment by rossie70** [comment]
@john-thomas-dotcms  We figured out how to reproduce it. Basically, if you use the Chrome Web Developer option to populate form fields and then check one or more boxes, dotCMS saves it as though they’re all checked, even if they aren’t.
